src/index.ts

```diff
@@ -36,3 +36,5 @@ export {
   NaivePaymaster__factory as NaivePaymasterFactory,
   UpgradableDocumentStore__factory as UpgradableDocumentStoreFactory,
 } from "./contracts";
+
+export * from "./contracts";
```

Comment:
Hum is this needed? We have renamed some export, this will lead to some inconsistency

Renaming happened after a breaking change in typechain (just for info), to. avoid a breaking in this library

Maybe there is a way to configure how factories are generated?

Reply:
I've tried to checked on this one but it doesn't look like there is a direct way to configure how the factories are being generated. 

Did you mean using the default exported names from `./contracts` broke something in this library and that was why we renamed the typechain factories before exporting?

Reply:
I wrote a little script to automate the manual process of renaming the factories from typechain on build. Hopefully this will resolve the issue with inconsistency. https://github.com/Open-Attestation/document-store/pull/99/commits/d2881aa3c76d091525eaa70a3cf87b1b2b55c929

Reply:
If this is okay, then I will merge this PR and release the fix today.

Reply:
Where is the node script called?

Reply:
ok nvm, the post thing =)

Reply:
yup, it's through the post thing.😁